src/unity/toolkits/one_shot_object_detection/one_shot_object_detector.hpp

```diff
@@ -0,0 +1,82 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#ifndef TURI_ONE_SHOT_OBJECT_DETECTOR_H_
+#define TURI_ONE_SHOT_OBJECT_DETECTOR_H_
+
+// TODO: Clean up imports.
```

Comment:
Any reason not to do that now? Let's not import things you're not using

Reply:
Done.